### PR TITLE
Add Radio Buttons

### PR DESCRIPTION
**Pull request by Anita77777**
Hey, this is a mini PR for 2 Radio Buttons.

### CODE REVIEW

File: src/components/Button/RadioButtonstyled.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	display: flex;
+	justify-content: start;
+	border: none;
+`;
+
+export const LabelRadio = styled.label`
+	display: flex;
+	padding: 20px;
+	color: white;
+`;
```
Comment (doemser) [inline src/components/Button/RadioButtonstyled.js:13]: try to put this components into separate files

**Review by doemser — CHANGES_REQUESTED** [review]
only a little change

File: src/components/Button/RadioButtonstyled.js (hunk 1 above)
Comment (Anita77777) [reply src/components/Button/RadioButtonstyled.js:13]: But they belong together, forever??

Comment (doemser) [reply src/components/Button/RadioButtonstyled.js:13]: chances are, that you want to reuse your styled.label at some point, but I will leave it up to you.

**Review by doemser — APPROVED** [review]
:)

**Pull request merged**